pkgs/tools/security/enpass/fet-urls.sh

```diff
@@ -0,0 +1,6 @@
+get() {
```

Comment:
the script currently fetches the apt metadata of that repository,
i'd like to factor it into a script that returns a nix dict with the real data